api/registration/models.py

```diff
@@ -32,13 +33,25 @@ class User(AbstractUser, PermissionsMixin):
     # Identify Information
     id = models.BigAutoField(primary_key=True)
     email = models.EmailField("Email Address", unique=True, blank=False, null=False)
+    
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = []
 
+    objects = UserManager()
+
+    @property
+    def full_name(self):
+        return '%s %s' % (self.first_name, self.last_name)
+
+class UserInformation(models.Model):
+    
     # Personal Information
+    UserId = models.BigAutoField(primary_key=True)
```

Comment:
If you checked Trello, you will see that `UserId` is not a primary key. It's a foreign key to link the user model with this model.
Change it to be a foreign key for the user model and name it `user`

Reply:
remember to take adjust the `UserInformation` model to be deleted if the user is deleted